config/client.go

```diff
@@ -132,6 +132,7 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["PasswordRequireNumber"] = strconv.FormatBool(*c.PasswordSettings.Number)
 	props["PasswordRequireSymbol"] = strconv.FormatBool(*c.PasswordSettings.Symbol)
 	props["CustomUrlSchemes"] = strings.Join(c.DisplaySettings.CustomUrlSchemes, ",")
+	props["IsDefaultMarketplace"] = strconv.FormatBool(*c.PluginSettings.MarketplaceUrl == model.PLUGIN_SETTINGS_DEFAULT_MARKETPLACE_URL)
```

Comment:
Not sure if we need to check for `PLUGIN_SETTINGS_OLD_MARKETPLACE_URL` for the initial 5.16 versions?

Reply:
Good thought! This shouldn't be necessary, since we forcibly upgrade the config to point at the new marketplace url if we find the old value.